### PR TITLE
Update timetravel.md

### PR DESCRIPTION
**Pull request by arthuo**
Overwrite with new ones_zh

### CODE REVIEW

File: site/zh-CN/userGuide/search/timetravel.md [hunk 1]
```diff
@@ -213,9 +213,9 @@ Milvus timestamp:           430390435713122310
 --------------------------  ------------------
 ```
 
-## Search with a specified timestamp
+## 使用指定的时间戳搜索
 
-Load the collection and search the target data with the timestamp of the first data batch. With the timestamp specified, Milvus only retrieves the data view at the point of time the timestamp indicates.
+加载 collection 并使用第一个数据批次的时间戳搜索目标数据。指定时间戳后，Milvus 只会在时间戳指示的时间点检索数据视图。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/timetravel.md:218]: 第一批数据

File: site/zh-CN/userGuide/search/timetravel.md [hunk 2]
```diff
@@ -414,13 +414,13 @@ No.1:
...
 ```
 
-## Generate a timestamp for search
+## Generate a timestamp for search为搜索生成时间戳
 
-In the case that the previous timestamp is not recorded, Milvus allows you to generate a timestamp using an existing timestamp, Unix Epoch time, or date time.
+在没有记录之前的时间戳的情况下，Milvus 允许你使用现有的时间戳、Unix Epoch 时间或日期时间生成时间戳。
 
-The following example simulates an unwanted deletion operation and shows how to generate a timestamp prior to the deletion and search with it.
+下面的示例模拟了不需要的删除操作，并展示如何在删除之前生成时间戳并使用它进行搜索。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/timetravel.md:421]: 模拟了意外的删除操作，并展示如何生成删除操作前的时间戳并使用它进行搜索

File: site/zh-CN/userGuide/search/timetravel.md [hunk 3]
```diff
@@ -560,7 +560,7 @@ No.1:
 +---------+------+------------+----------+
 ```
 
-Search with the prior-to-deletion timestamp. Milvus retrieves entities from the data before the deletion.
+使用删除之前的时间戳进行搜索。 Milvus 在删除之前从数据中检索 entity。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/timetravel.md:563]: Milvus 在删除操作之前的数据试图中检索 entity。

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by LocoRichard** [comment]
Greetings, @arthuo. Thank you! I have made a few comments.

How is your research btw? I hope it is going smoothly.

**Comment by arthuo** [comment]
> Greetings, @arthuo. Thank you! I have made a few comments.
> 
> How is your research btw? I hope it is going smoothly.

Glad to receive your feedback. I think what you pointed out is very good and takes my understanding of Milvus one step further. Also, my recent research is going well, thank you for your concern.

**Pull request merged**